scipy/stats/_morestats.py

```diff
@@ -2645,6 +2646,8 @@ def levene(*samples, center='median', proportiontocut=0.05):
     return LeveneResult(W, pval)
 
 
+@_deprecated("'binom_test' is deprecated in favour of"
+             " 'binomtest' and will be removed in Scipy 1.11.0.")
```

Comment:
Personally when reading deprecation message, the critical information is since when the alternative is present – not when the current one will be removed. From my understanding this means `binomtest` is available since 0.19.0, so I would state that in the deprecation message as well. 

Knowing since when it is available is often the difference between understanding whether you can do a blind rename, of have to do conditional based on version number.

Just my 2c.